js_modules/dagster-ui/packages/ui-components/src/components/CollapsibleSection.tsx

```diff
@@ -5,20 +5,22 @@ import {Icon} from './Icon';
 
 export const CollapsibleSection = ({
   header,
+  containerProps,
   headerWrapperProps,
   children,
   isInitiallyCollapsed = false,
   arrowSide = 'left',
 }: {
   header: React.ReactNode;
+  containerProps?: React.ComponentProps<typeof Box>;
   headerWrapperProps?: React.ComponentProps<typeof Box>;
   children: React.ReactNode;
   isInitiallyCollapsed?: boolean;
   arrowSide?: 'left' | 'right';
 }) => {
   const [isCollapsed, setIsCollapsed] = React.useState(isInitiallyCollapsed);
   return (
-    <Box flex={{direction: 'column'}}>
```

Comment:
Generally, rather than having a `containerProps` prop, I prefer using `styled-components` to create a Custom wrapper element, then using styled-components from the parent component to target that custom element. Generally I find it more flexible and it avoids prop drilling.

Reply:
Ahh interesting, I always found the pattern of creating a component just to modify the appearance of a different component sorta workaroundish -- I think this will be a non-issue if I apply Dish's suggestion to just rework this dialog into tabs instead of vertically split lists, so we don't have to figure it out 😅